go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/foundriesio/fioconfig
 
-go 1.17
```

Comment:
Probably this commit should be extracted into the separate PR?

Reply:
ok